src/main/asciidoc/is-new-state-detection.adoc

```diff
@@ -7,14 +7,14 @@ The following table describes the strategies that Spring Data offers for detecti
 [options = "autowidth",cols="1,1"]
 |===
 |`@Id`-Property inspection (the default)
-|By default, Spring Data inspects the version property of the given entity.
-If the identifier property is `null` or `0` in case of primitive types, then the entity is assumed to be new.
+|By default, Spring Data inspects the id property of the given entity.
+If the id property is `null` or `0` in case of primitive types, then the entity is assumed to be new.
```

Comment:
Any reason you're changing `identifier` into `id` in random places?

Reply:
In some places in the text there is Id, in other places there is an identifier. I thought I would unify it in this section. I don't care whether it's id or identifier, I just thought id is closer to @Id than identifier.